python/port/mod/turtle/turtle.h

```diff
@@ -76,12 +79,26 @@ class Turtle {
   static constexpr int k_invertedYAxisCoefficient = -1;
   static constexpr KDCoordinate k_xOffset = Ion::Display::Width / 2;
   static constexpr KDCoordinate k_yOffset = (Ion::Display::Height - Metric::TitleBarHeight) / 2;
-  static constexpr int k_numberOfIcons = 8;
   static constexpr uint8_t k_defaultSpeed = 3;
   static constexpr uint8_t k_maxSpeed = 10;
   static constexpr KDColor k_defaultColor = KDColorBlack;
   static constexpr uint8_t k_defaultPenSize = 1;
 
+  enum class PawType : uint8_t {
+    FrontRight = 0,
+    BackRight = 1,
+    FrontLeft = 2,
+    BackLeft = 3
+  };
+
+  enum class PawPosition : int {
```

Comment:
Why bother specifying `int` as a backing type?

Reply:
I forgot to clean this ! Done with 91c4fea